.editorconfig

```diff
@@ -5,3 +5,6 @@ indent_size = 4
 tab_width = 4
 trim_trailing_whitespace = true
 indent_style = tab
+
+dfmt_brace_style=allman
+dfmt_space_after_cast=false
```

Comment:
Have I been forgetting spaces after casts? (Not that this matters. dfmt doesn't do anything with this option yet.)